app/views/settings/_min_width_sort_criteria.html.erb

```diff
@@ -0,0 +1,162 @@
+<% columns = IssueQuery.new.available_inline_columns %>
+<% column_captions = columns.each_with_object({}) do |column, hash|
+     hash[column.name.to_s] = column.caption
+   end %>
+
+<% #Retrieve settings %>
+<% settings = @settings || [] %>
+<% column_settings = settings.find { |setting| setting[0] == "issue_list_defaults" } %>
+<% column_names = column_settings&.dig(1, "column_names") || [] %>
+<% columns_sorting = settings["columns_sorting"] || "" %>
+<% columns_min_width = settings["columns_min_width"] || "" %>
+
+<% #Re-arrange column_names based on sorting order %>
+<% sorting_columns = columns_sorting.split(",").map { |criterion| criterion.split(":").first.strip } %>
+<% column_names = column_names = sorting_columns + (column_names - sorting_columns) %>
+
+<fieldset class="box" id="sorting_criteria_box">
+  <legend><%= l :label_sorting_columns %></legend>
+  <% selected_tag_id = "sorting_columns" %>
+  <% tag_name = "columns[]" %>
+
+  <div class="buttons" style="display:grid">
+    <input type="button" value="⇈" onclick="moveOptionss('top');" />
+    <input type="button" value="↑" onclick="moveOptionss('up');" />
+    <input type="button" value="↓" onclick="moveOptionss('down');" />
+    <input type="button" value="⇊" onclick="moveOptionss('bottom');" />
+  </div>
+
+  <div class="query-columns">
+    <p style="padding-left:0"><%= l(:label_sorting_priority_columns) %></p>
+    <%= select_tag tag_name,
+                   options_for_select(column_names.map { |name| [column_captions[name], name] }),
+                   id: selected_tag_id,
+                   multiple: true,
+                   size: 12,
+                   class: "capitalize-text" %>
+  </div>
+
+  <div>
+    <div id="input_header"><h3>Column name</h3><h3>Sorting order</h3><h3>Min width</h3></div>

```

Comment:
You should also internationalize this. Anything that is a static lable should be internationalized.